test/e2e/rollout_test.go

```diff
@@ -542,6 +548,231 @@ var _ = Describe("placing wrapped resources using a CRP", Ordered, func() {
+			crp.Spec.Policy = &placementv1beta1.PlacementPolicy{
+				PlacementType: placementv1beta1.PickFixedPlacementType,
+				ClusterNames: []string{
+					memberCluster1EastProdName,
+				},
+			}
+			crp.Spec.Strategy.RollingUpdate.UnavailablePeriodSeconds = ptr.To(60)
+			Expect(hubClient.Create(ctx, crp)).To(Succeed(), "Failed to create CRP")
+		})
+
+		It("should update CRP status as expected", func() {
+			crpStatusUpdatedActual := customizedCRPStatusUpdatedActual(crpName, wantSelectedResources, []string{memberCluster1EastProdName}, nil, "0", false)
+			Eventually(crpStatusUpdatedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update CRP status as expected")
+		})
+
+		It("should place the resources on member cluster", func() {
+			workResourcesPlacedActual := waitForTestResourceToBePlaced(memberCluster1EastProd, &testCustomResource)
+			Eventually(workResourcesPlacedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to place work resources on member cluster %s", memberCluster1EastProd.ClusterName)
+		})
+
+		It("should update custom resource", func() {
+			Eventually(func() error {
+				var cr testv1alpha1.TestResource
+				err := hubClient.Get(ctx, types.NamespacedName{Name: testCustomResource.Name, Namespace: workNamespace.Name}, &cr)
+				if err != nil {
+					return err
+				}
+				cr.Spec.Foo = "bar1" // Previously was "foo1"
+				return hubClient.Update(ctx, &cr)
+			}, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update custom resource")
+		})
+
+		It("should not update the resource on member cluster", func() {
+			// subtracting a second because transition between IT takes ~1 second
+			unavailablePeriod := time.Duration(*crp.Spec.Strategy.RollingUpdate.UnavailablePeriodSeconds)*time.Second - time.Second
+			Consistently(func() error {
+				var cr testv1alpha1.TestResource
+				err := hubClient.Get(ctx, types.NamespacedName{Name: testCustomResource.Name, Namespace: workNamespace.Name}, &cr)
+				if err != nil {
+					return err
+				}
+				if cr.Spec.Foo != "bar1" { // Previously was "foo1"
+					return nil
+				}
+				return err // Other errors should be returned
+			}, unavailablePeriod, consistentlyInterval).Should(BeNil(), "Test resource was updated when it shouldn't be")
+		})
+
+		It("should update CRP status as expected", func() {
+			crpStatusUpdatedActual := customizedCRPStatusUpdatedActual(crpName, wantSelectedResources, []string{memberCluster1EastProdName}, nil, "1", false)
+			Eventually(crpStatusUpdatedActual, longEventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update CRP status as expected")
+		})
+
+		AfterAll(func() {
+			// Remove the custom deletion blocker finalizer from the CRP.
+			ensureCRPAndRelatedResourcesDeletion(crpName, []*framework.Cluster{memberCluster1EastProd})
+		})
+	})
+
+	Context("Test a CRP place workload objects successfully, should not wait to place/update resource", Ordered, func() {
```

Comment:
I am not sure what this case is testing

Reply:
Testing that there should not be a wait time since these will be ready and available as soon as they are created.

Reply:
I guess this test mostly tests the calculating the wait time more than this PR.